app/src/common/chromium/com/igalia/wolvic/browser/api/impl/SettingsImpl.java

```diff
@@ -88,8 +91,7 @@ public void setViewportMode(int value) {
 
     @Override
     public int getUserAgentMode() {
-        // TODO: implement
-        return WSessionSettings.USER_AGENT_MODE_MOBILE;
+        return mSessionSettings.getUserAgentMode();
```

Comment:
Ditto, here we would convert from chromium backend values to wolvic values.

Reply:
Fixed.